en/docs/guides/login/saml-parameters-in-auth-request.md

```diff
@@ -45,8 +45,7 @@ https://localhost:9443/samlsso?spEntityID=foo.com&tenantDomain=abc.com&forceAuth
 -----
 
 !!! info "Related Topics"
-    - [Concept: SAML](TODO:insert-link-to-concept)
+    - [Concept: SAML](../../../concepts/authentication/intro-saml)
     - [Guide: Single Sign-On](../enable-single-sign-on)
     - [Guide: Enable Login for a SAML Application](../webapp-saml)
     - [Guide: SAML Authentication Error Codes](TODO:insertlink)
-
```

Comment:
`- [Guide: SAML Authentication Error Codes](TODO:insertlink)` let's remove this as we currently don't have a doc for this